actors/builtin/miner/miner_actor.go

```diff
@@ -1363,24 +1363,33 @@ func (a Actor) ReportConsensusFault(rt Runtime, params *ReportConsensusFaultPara
 	}
 
 	// Reward reporter with a share of the miner's current balance.
-	slasherReward := RewardForConsensusSlashReport(faultAge, rt.CurrentBalance())
-	_, code := rt.Send(reporter, builtin.MethodSend, nil, slasherReward)
-	builtin.RequireSuccess(rt, code, "failed to reward reporter")
-
 	var st State
 	rt.State().Readonly(&st)
+	unlockedBalance := st.GetUnlockedBalance(rt.CurrentBalance())
+	slasherReward := RewardForConsensusSlashReport(faultAge, rt.CurrentBalance())
```

Comment:
I realise this was totally unaddressed by the issue, but I was expecting the amount of which the reporter earns a share to be the `ConsensusFaultPenalty`, not a function of the miner's size. So we'd compute the penalty, then split it into reward share and burn share. `ConsensusFaultPenalty` is the maximum total lost by the miner.

@zixuanzh @nicola could you please confirm this interpretation?

If I'm wrong, and we do want to be taking the reward from the miner's balance over and above the consensus fault penalty, then we should only consider that part of the balance that is available for paying fees: the unlocked balance plus the vesting funds. I think it would follow the same logic as `PenalizeFundsInPriorityOrder`. The fact of a miner having a large pre-commit deposit lockup should not enter into it.

Reply:
Correct, only the consensus fault penalty is lost by the miner.

Reply:
One minor remaining problem is that now we are trying to pay an actual account for fees that could possibly only be covered by debt.  Fee debt is currently only repayable to the burnt funds actor.  Extending it to be repayable to any slashing account does not seem worth the state tracking cost.

Proposal: reward reporter with between `0` and `slasherReward` FIL (slasherReward when the penalty does not include any fee debt, 0 when the penalty is totally covered by fee debt).  The reporter never gets paid back out of fee debt.

Reply:
FYI @nicola ^